source/TLS.rst

```diff
@@ -27,40 +28,63 @@ Operator yourself, as well as how to temporarily disable it if needed.
+The `cert-manager <https://cert-manager.io/docs/>`_ is a Kubernetes certificate
+management controller which widely used to automate the management and issuance
+of TLS certificates. It is community-driven, and open source. 
 
-When you have already installed *cert-manager* and deploy the operator, the operator requests a certificate from the *cert-manager*. The *cert-manager* acts as a self-signed issuer and generates certificates. The Percona Operator self-signed issuer is local to the operator namespace. This self-signed issuer is created because Percona Server for MongoDB requires all certificates are issued by the same CA.
+When you have already installed *cert-manager* and deploy the operator, the
+operator requests a certificate from the *cert-manager*. The *cert-manager* acts
+as a self-signed issuer and generates certificates. The Percona Operator
+self-signed issuer is local to the operator namespace. This self-signed issuer
+is created because Percona Server for MongoDB requires all certificates issued
+by the same :abbr:`CA (Certificate authority)`.
 
-The creation of the self-signed issuer allows you to deploy and use the Percona Operator without creating a clusterissuer separately.
+Self-signed issuer allows you to deploy and use the Percona
+Operator without creating a cluster issuer separately.
 
 Installation of the *cert-manager*
 ----------------------------------
 
 The steps to install the *cert-manager* are the following:
 
-* Create a namespace
-* Disable resource validations on the cert-manager namespace
-* Install the cert-manager.
+* create a namespace,
+* disable resource validations on the cert-manager namespace,
+* install the cert-manager.
 
 The following commands perform all the needed actions:
 
 .. code:: bash
 
-   $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml --validate=false
+   $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v{{{certmanagerrecommended}}}/cert-manager.yaml --validate=false
 
-After the installation, you can verify the *cert-manager* by running the following command:
+After the installation, you can verify the *cert-manager* by running the
+following command:
 
 .. code:: bash
 
    $ kubectl get pods -n cert-manager
 
-The result should display the *cert-manager* and webhook active and running.
+The result should display the *cert-manager* and webhook active and running:
+
+.. code:: text
+
+   NAME                                       READY   STATUS    RESTARTS   AGE
+   cert-manager-7d59dd4888-tmjqq              1/1     Running   0          3m8s
+   cert-manager-cainjector-85899d45d9-8ncw9   1/1     Running   0          3m8s
+   cert-manager-webhook-84fcdcd5d-697k4       1/1     Running   0          3m8s
+
+Now, whenever you :ref:`check certificates for expiration<tls.certs.update.check>`,
+you will find that they are valid and short-term.
```

Comment:
```suggestion
Once you create the database with the Operator, it will automatically trigger 
cert-manager to create certificates. Whenever you :ref:`check certificates for expiration<tls.certs.update.check>`,
you will find that they are valid and short-term.
```